### PR TITLE
Use stable v17.12 Docker dind image.

### PR DESCRIPTION
**Pull request by bweston92**
Uses the latest stable dind by default, which at the minute has better default ca certificates.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # docker build --rm -f docker/Dockerfile -t plugins/docker .
 
-FROM docker:17.10.0-ce-dind
+FROM docker:stable-dind
```
Comment (bradrydzewski) [inline docker/Dockerfile]: We pin the plugin to a specific docker version and tag the plugin using the same version number as Docker. So we would accept `17.12.0-ce-dind` which is the same as `stable-dind` with a version bump in the `.drone.yml` for `17.12`

Comment (bweston92) [reply docker/Dockerfile]: @bradrydzewski thanks for the quick response, updated the PR.

File: .drone.yml [hunk 2]
```diff
@@ -23,7 +23,7 @@ pipeline:
   publish_heroku:
     image: plugins/docker
     repo: plugins/heroku
-    tags: [ "latest", "17", "17.10" ]
+    tags: [ "latest", "17", "17.10", "17.12" ]
```
Comment (bradrydzewski) [inline .drone.yml]: remove 17.10

```diff
-tags: [ "latest", "17", "17.10", "17.12" ]
+tags: [ "latest", "17", "17.12" ]
```

File: .drone.yml [hunk 3]
```diff
@@ -13,7 +13,7 @@ pipeline:
   publish:
     image: plugins/docker
     repo: plugins/docker
-    tags: [ "latest", "17", "17.10" ]
+    tags: [ "latest", "17", "17.10", "17.12" ]
```
Comment (bradrydzewski) [inline .drone.yml]: remove 17.10

```diff
-tags: [ "latest", "17", "17.10", "17.12" ]
+tags: [ "latest", "17", "17.12" ]
```

**Comment by bradrydzewski** [comment]
thanks for the changes. 1 minor fix in the yaml, and I'll get it merged right away.

**Pull request merged**